qualang_tools/bakery/bakery.py

```diff
@@ -9,20 +9,50 @@
 import copy
 
 
-def baking(config, padding_method="right"):
-    return Baking(config, padding_method)
+def baking(
+    config,
+    padding_method="right",
+    override=False,
+    update_config: bool = True,
+    baking_index: int = None,
+):
+    """
+    Opens a context manager to synthesize samples for arbitrary waveforms
+    :param config: config file
+    :param padding_method: Method to pad 0s to format the waveform to match hardware constraint
+    (>16 ns and multiple of 4)
+    :param override: Define if baked waveforms are overridable when using add_compiled feature
+    :param update_config: Define if baked waveform should be added within the input config file
```

Comment:
What is the alternative?

Reply:
Read about it in the readme. Add here that the default is true.

Is there any usecase in which this would be False but the baking_index won't be provided?

If the answer is no, then it might be easier to just get rid of this parameter...